README.md

```diff
@@ -0,0 +1,9 @@
+# alex-cio.de
+
+Hello my friends, 
+thanks you for your contributions. 
+If you need more help, send me a message mailto:info@alex-cio.de
+
+<a href="https://twitter.com/alexcio_">Follow me on Twitter</a>
```

Comment:
try using markdown syntax `[I'm an inline-style link](https://www.google.com)`

Reply:
Done it!